node/client/src/lib.rs

```diff
@@ -223,11 +223,26 @@ impl Client {
              This should never happen, because block production is atomic."
         );
 
+        info!(target: "client", "Producing block index: {:?}, beacon hash = {:?}, shard hash = {:?}",
+            block.body.header.index,
+            block.hash,
+            shard_block.hash,
+        );
+        let block_receipts = get_all_receipts(shard_block.body.receipts.iter());
```

Comment:
Wrap this code into https://docs.rs/log/0.4.6/log/macro.log_enabled.html

Reply:
Good point